Sources/OpenCombine/Publishers/Publishers.Encode.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  Publishers.Encode.swift
+//  
+//
+//  Created by Joseph Spadafora on 6/22/19.
+//
+
+
+extension Publishers {
+    
+    public struct Encode<Upstream, Coder> : Publisher where Upstream : Publisher, Coder : TopLevelEncoder, Upstream.Output : Encodable {
```

Comment:
All that I've mentioned for `Decode` applies here as well, I guess

Reply:
Fixed